cdc/capture/capture.go

```diff
@@ -233,9 +229,8 @@ func (c *captureImpl) reset(ctx context.Context) error {
 		c.sorterSystem = ssystem.NewSystem(sortDir, memPercentage, c.config.Debug.DB)
 		err = c.sorterSystem.Start(ctx)
 		if err != nil {
-			return errors.Annotate(
-				cerror.WrapError(cerror.ErrNewCaptureFailed, err),
-				"create sorter system")
+			log.Error("start sorter system failed", zap.Error(err))
```

Comment:
Some errors have been printed in the log and some of them don't output the logs. Can we unify the behavior?